frontend/cmd/cmd.go

```diff
@@ -38,13 +38,16 @@ import (
 	"github.com/Azure/ARO-HCP/frontend/pkg/frontend"
 	"github.com/Azure/ARO-HCP/frontend/pkg/util"
 	"github.com/Azure/ARO-HCP/internal/api/arm"
+	"github.com/Azure/ARO-HCP/internal/audit"
 	"github.com/Azure/ARO-HCP/internal/database"
 	"github.com/Azure/ARO-HCP/internal/ocm"
 	"github.com/Azure/ARO-HCP/internal/tracing"
 	"github.com/Azure/ARO-HCP/internal/version"
 )
 
 type FrontendOpts struct {
+	auditNoop bool
```

Comment:
```suggestion
	auditNoOp bool
```
I think this casing is clearer. You actually do it like this in initialiser function name